applications/nrf_desktop/README.rst

```diff
@@ -1521,6 +1521,19 @@ These are valid for events that have many listeners or sources, and are shared f
 
    doc/event_rel_modules.rst
 
+.. _nrf_desktop_known_issues:
+
+Known issues
+************
+
+* Boot reports are not sent
+* Dongle - Keyboard connection deteriorates badly with split_ll
+* Unable to re-bond keyboard with OSX host
+* Dongle unable to setup connection with Desktop peripherals
+* Bond switching in Gaming Mouse stops working
+* Config channel should use write without response
```

Comment:
Is this an issue or a planned improvement?
Is the issue that the config channel is slow, that it affects the report rate or both.